src/pages/EmployeesPage.tsx

```diff
@@ -0,0 +1,174 @@
+            className="sm:hidden bg-gray-200 px-3 py-2 rounded"
+            onClick={() => setFilters(true)}
+          >
+            Filters
+          </button>
+          <div className="hidden sm:flex gap-4">
+            <FilterDropdown
+              label={"Department"}
+              options={departments}
+              value={department}
+              onChange={setDepartment}
+            />
+            <FilterDropdown
+              label={"Role"}
+              options={roles}
+              value={role}
+              onChange={setRole}
+            />
+            <FilterDropdown
+              label={"Status"}
+              options={employmentStatuses}
+              value={status}
+              onChange={setStatus}
+            />
+            <FilterDropdown
+              label={"Contract"}
+              options={contractTypes}
+              value={contractType}
+              onChange={setContractType}
+            />
+          </div>
+          {filters && (
+            <div className="fixed inset-0 bg-black/80 flex justify-center items-center z-50">
+              <div className="bg-white rounded-lg p-6 w-80">
+                <h2 className="text-lg font-bold mb-4">Filters</h2>
+                <div className="space-y-4">
+                  <FilterDropdown
+                    label="Department"
+                    options={departments}
+                    value={department}
+                    onChange={setDepartment}
+                  />
+                  <FilterDropdown
+                    label="Role"
+                    options={roles}
+                    value={role}
+                    onChange={setRole}
+                  />
+                  <FilterDropdown
+                    label="Status"
+                    options={employmentStatuses}
+                    value={status}
+                    onChange={setStatus}
+                  />
+                  <FilterDropdown
+                    label="Contract"
+                    options={contractTypes}
+                    value={contractType}
+                    onChange={setContractType}
+                  />
+                </div>
+                <div className="flex justify-end mt-6">
+                  <button
+                    className="bg-blue-600 text-white px-4 py-2 rounded"
+                    onClick={() => setFilters(false)}
+                  >
+                    Apply
+                  </button>
+                </div>
+              </div>
+            </div>
+          )}
+        </div>
+      </div>
+      <div className="grid gap-4 sm:grid-cols-2 lg:grid-cols-3">
+        {filteredEmployees.map((emp) => (
+          <EmployeeCard key={emp.id} employee={emp} onEdit={(id) => {
+            const found = employeeList.find((e) => e.id === id);
+            if (found) setEditEmployee(found);
+          }}
+          onDelete={handleDelete}
+          />
+        ))}
+      </div>
+
+      {addEmployee && (
```

Comment:
There is a better was to manage modals, I noticed that your modals dont have entry and exit animations. You might want to look at framer motion or react transition component, or you can build a custom modal component with very good entry and exit animations